src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c

```diff
@@ -27,6 +48,12 @@ const ActorInit Obj_Tsubo_InitVars = {
     NULL,
 };
 
+static s16 sObjectIds[] = { OBJECT_GAMEPLAY_DANGEON_KEEP, OBJECT_TSUBO };
+
+static Gfx* D_80BA1B84[] = { 0x05017870, 0x060017C0 };
+
+static Gfx* D_80BA1B8C[] = { 0x05017A60, 0x06001960 };
```

Comment:
these 05 segmented addresses are from gameplay_dangeon_keep, which is decompiled already. so youll want to include that header here and use those symbols.
its a bit confusing to know when dangeon_keep and field_keep are used since they share the same segment number. but i determined that by the array above that has dangeon_keep in it as a possible object id

Reply:
the two symbols that start with 0x05 there are `gPotDL` and `gPotFragmentDL` respectively. I searched for the offsets contained in the segmented addresses in the xml file for gameplay_dangeon_keep

If any of that is not clear how I determined it tho feel free to ask questions

Reply:
I did try using gPotDL and gPotFragmentDL, but it doesn't provide a match if I use them (I used tubo trap as a reference since it is almost identical). Does using them require running make clean && make setup?

Reply:
it shouldnt, no. can you post here or in discord what the diff is?

Reply:
though to be safe you can do those things. but if it works for tubo_trap it will work here as well